cartoframes/data/observatory/repository/category_repo.py

```diff
@@ -0,0 +1,41 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_category_repo():
+    return CategoryRepository()
```

Comment:
In case we decide to access the Repository classes through this method instead of having module-level instances, shouldn't we create the instance once instead of each time this method is called? Also, see comment about singleton patter in the repo_client file.

Reply:
Yes, I'd say we could just have one instance of the repository.

Reply:
OK, I'll change it in the next issue.